README.md

```diff
@@ -0,0 +1,29 @@
+# Discord
+
+Discord is a free all-in-one messaging, voice and video client that's available on your computer and phone.
+
+This repo hosts the flatpak wrapper for [Discord](https://discord.com/), available at [Flathub](https://flathub.org/apps/details/com.discordapp.Discord).
+
+```sh
+flatpak install flathub com.discordapp.Discord
+flatpak run com.discordapp.Discord
+```
+
+
+## Differences in flatpak version
+
+The flatpak version runs in a sandbox to provide better safety and privacy for users (see [this privacy report](https://spyware.neocities.org/articles/discord.html)).
```

Comment:
This feels inappropriate to put in a page which is supposed to represent the application.

Reply:
```suggestion
The flatpak version runs in a sandbox to provide better safety and privacy for users.
```